lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -179,9 +191,30 @@ export default function BasicLtiLaunchApp() {
   }, [authToken, authUrl, fetchContentUrl, lmsName]);
 
   if (ltiLaunchState.state === 'fetched-url') {
-    return (
-      <iframe width="100%" height="100%" src={ltiLaunchState.contentUrl} />
+    const iFrame = (
+      <iframe
+        key={focusedUser}
+        width="100%"
+        height="100%"
+        className="js-via-iframe"
+        src={ltiLaunchState.contentUrl}
+      />
     );
+
+    if (lmsGrader) {
+      // Use the LMS Grader
+      return (
+        <LMSGrader
+          onChangeUser={username => {
+            setFocusedUser(username);
+          }}
+        >
+          {iFrame}
```

Comment:
I gave this a try and I'd rather leave it nested. For one thing, when its nested, the sidebar reacts to the grader's block content and automatically gets placed below it because the <ul> is its sibling. If we move the iframe outside of this component it renders under the grader and is hidden.

![Screen Shot 2019-09-20 at 5 24 24 PM](https://user-images.githubusercontent.com/3939074/65365125-88b63f80-dbcb-11e9-9854-f9b79cb50310.png)


Id rather not do a fixed padding or anything. But I think the dOM didn't make sense as it was. It makes it more of a symbolic dom element now, but has good readability.

![Screen Shot 2019-09-20 at 5 23 25 PM](https://user-images.githubusercontent.com/3939074/65365103-5f95af00-dbcb-11e9-9948-c1c5c79ddf00.png)